skymp5-server/cpp/server_guest_lib/GetBaseActorValues.h

```diff
@@ -13,6 +13,10 @@ struct BaseActorValues
   float staminaRateMult = 100;
   float magickaRateMult = 100;
 
+  float healthPercentage = 1.0f;
+  float magickaPercentage = 1.0f;
+  float staminaPercentage = 1.0f;
+
```

Comment:
I'm not sure if this is the solution that was required - it might be reasonable to keep the value higher and only transform the data output.